xknx/devices/scene.py

```diff
@@ -42,6 +42,11 @@ def _iter_remote_values(self) -> Iterator[RemoteValueSceneNumber]:
         """Iterate the devices RemoteValue classes."""
         yield self.scene_value
 
+    @property
+    def unique_id(self) -> Optional[str]:
+        """Return unique id for this device."""
+        return f"{self.scene_value.group_address}"
```

Comment:
we should add the scene number as a scene Entity is bound to a scene number and a GA. One GA can have multiple scenes.